### PR TITLE
#169257392 Set up Docker

### PR DESCRIPTION
**Pull request by Mastel22**
#### What does this PR do?
Sets up Docker
#### Description of Task to be completed?

- [X] Added Dockerfile
- [X] Added .dockerignore
- [X] Added docker-compose.yml

#### How should this be manually tested?

1. Install Docker on your pc following instructions [here](https://docs.docker.com/docker-for-mac/install/)

2. Git clone this repo 

3. While on your project root: - run `docker-compose up --build`
                                                  - run `docker images` to make sure the image wa created
4. At successful creation of the image run `docker ps` get the container ID

5. Run docker exec it `container id` sh

6. Run `npm run migrate` 

7. Use Postman to test endpoint

#### Any background context you want to provide?
-N/A
#### What are the relevant pivotal tracker stories?
[#169257392](https://www.pivotaltracker.com/story/show/169257392)
#### Screenshot
None
#### Questions:
Can't one just push the image to the hub so that anyone can pull it then work with that?

### CODE REVIEW

**Review by timolinn — COMMENTED** [review]
Can we put a hold on this feature till next week? We'll come back to this once we have a functional base application.

**Comment by timolinn** [comment]
Closing it now, will reopen when base app is functional.

File: package.json [hunk 1]
```diff
@@ -31,6 +31,12 @@
     "bcrypt": "^3.0.6",
     "body-parser": "^1.19.0",
     "cors": "^2.8.4",
+    "@babel/cli": "^7.6.4",
+    "@babel/core": "^7.6.4",
+    "@babel/node": "^7.6.3",
+    "@babel/preset-env": "^7.6.3",
+    "@babel/register": "^7.6.2",
+    "@babel/polyfill": "^7.6.0",
     "dotenv": "^6.0.0",
```
Comment (b0nbon1) [inline package.json]: Great work but why did you move babel to the dependencies?

**Review by timolinn — CHANGES_REQUESTED** [review]
Please add documentation for running and setup. Do this in the `README.md` file.

**Comment by Mastel22** [comment]
> Please add documentation for running and setup. Do this in the `README.md` file.



> Please add documentation for running and setup. Do this in the `README.md` file.

Yes, sure

**Review by timolinn — DISMISSED** [review]

**Review by b0nbon1 — DISMISSED** [review]
Looks good to me

**Review by gildniy — CHANGES_REQUESTED** [review]
Looks good, but I think by `.docker-compose` in the PR description you meant `docker-compose.yml` in the commit.

**Comment by Mastel22** [comment]
> I think by `.docker-compose` in the PR description you meant `docker-compose.yml` in the commit.

Oh yeah.Resolved

**Review by gildniy — APPROVED** [review]
Looks good to be merged!

**Review by b0nbon1 — APPROVED** [review]
Good work.

**Review by timolinn — APPROVED** [review]

**Pull request merged**